routing_common/transit_types.hpp

```diff
@@ -30,6 +33,7 @@ ShapeId constexpr kInvalidShapeId = std::numeric_limits<ShapeId>::max();
 // Note. Weight may be a default param at json. The default value should be saved as uint32_t in mwm anyway.
 // To convert double to uint32_t at better accuracy |kInvalidWeight| should be close to real weight.
 Weight constexpr kInvalidWeight = -1.0;
+Anchor constexpr kInvalidAnchor = std::numeric_limits<Anchor>::max();
```

Comment:
I dislike the fact that Anchor here and Anchor in drape are different data types, but you mix them.

Reply:
I guess, it's possible to move enum Anchor out of drape to platform and use it in drape and in routing_common. 

@rokuz @darina What do you think about that?

Reply:
As we agreed, nothing will be moved from drape.

Reply:
After discussion we decided that there're two different types: one for saving info in mwm. Another one for drape. It won't be good if possible future changes in drape will brake compatibility with mwm. So I'm returning number of type transit::Anchor to mwm tests.